src/aiida_qe_xspec/workflows/xps.py

```diff
@@ -93,15 +189,6 @@ def define(cls, spec):
                 ' element. Must use the mapping "{element}" : {Upf}".'
             )
         )
-        spec.input_namespace(
-            'gipaw_pseudos',
-            valid_type=(orm.UpfData, UpfData),
-            dynamic=True,
-            help=(
-                'Dynamic namespace for pairs of ground-state pseudopotentials for each absorbing'
-                ' element. Must use the mapping "{element}" : {Upf}".'
-            )
-        )
```

Comment:
Since this part has been removed, and the `gipaw_pseudos` has been merged into `core_hole_pseudos`, the `help` string for `core_hole_pseudos` needs to be updated for the new input format.

Doesn't need to be extensive, just something like:
```
'Dynamic namespace for ground-state and core-hole pseudopotentials for each absorbing'
' element. Must use the mapping "{element}" : {"gipaw" : {Upf}, "1s" : {Upf}}".'

Reply:
Done, in commit: [3b52fcb](https://github.com/aiidaplugins/aiida-qe-xspec/pull/16/commits/3b52fcb9fb1aa239342b5f9bc8858e9375c87d55)